### PR TITLE
PR fixing issue #162

### LINKED ISSUE #162: Store `current_log` in local storage for navigation points to `daily.html` from `weekly.html`
When `daily.html` is navigated to from the monthly/weekly view or the index, we must use additional context in order to determine which daily log should be requested from indexDB. With regards to the finalized schema, we are using a field `current_log`, which takes a UNIX timestamp as a string. This field can be used to determine the daily log that should be sent from indexDB when `daily.html` is loaded in the DOM. We outline this task as follows:
1. From `weekly.html`, when any day is clicked, `current_log` should be given a UNIX timestamp corresponding to the current day. The appropriate UNIX timestamp will be stored in the corresponding anchor `<a>` element as the data attribute `data-unix-timestamp`. More information about data attributes are found [here](https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes).
2. When `daily.html` is loaded, indexDB should be queried for a daily log with the same UNIX timestamp. If none are found, indexDB should store a new daily log for the given day. Otherwise, the existing entry should be sent upstream to the frontend.

### PR DESCRIPTION
**Pull request by brettherbst**
This PR updates the weekly.html hyperlinks, now clicking on a day with a daily log will take you to that day's daily log, and populates daily.html accordingly. Also updates the display of the specific day's calendar dates on weekly.html

### CODE REVIEW

File: source/html/daily.html [hunk 1]
```diff
@@ -20,8 +20,7 @@
       <div id="journal-content">
         <div id="title">
           <div class="date">
-            <!-- TODO: this needs to fetch today's date-->
-            <h1 class="date">May 24th</h1>
+            <h1 class="date"></h1>
```
Comment (yuzilyu) [inline source/html/daily.html:23]: For some reason, on my branch, the date is never fetched. Does that happen to you?

**Review by yuzilyu — APPROVED** [review]

**Comment by yuzilyu** [comment]
I can do the linting fix

File: source/html/daily.html (hunk 1 above)
Comment (NoahT) [reply source/html/daily.html:23]: @yuzilyu I was pair programming with Brett earlier; worked fine for us on his browser. Maybe add it as an issue?

Comment (brettherbst) [reply source/html/daily.html:23]: @yuzilyu  Hmmm, like Noah said it worked fine on my browser, may need to look into this

Comment (yuzilyu) [reply source/html/daily.html:23]: I think we should have e2e test cases for this

Comment (yuzilyu) [reply source/html/daily.html:23]: on my browser, it either displays nothing or 5/24/2021

**Pull request merged**